### DISCUSSION TITLE
How does one retrieve the optimal fit parameters from the AutoETS forecaster?

### DISCUSSION THREAD

**Discussion by ericjb** [alan-turing-institute/sktime#7184] [Q&A] [answered]
Suppose I want to get the best SES (Simple Exponential Smoothing) model for a time series. This involves finding the best initial level L0 and the best learning parameter \alpha (aka 'smoothing_level' in ExponentialSmoothing). 
This can be achieved via:

```
from sktime.forecasting.ets import AutoETS
forecaster = AutoETS(auto=False, trend=None, seasonal=None, n_jobs=-1)
```
or possibly

`forecaster = AutoETS(auto=False, trend=None, seasonal=None, return_params=True, n_jobs=-1).`

Then

`forecaster.fit(mydata)`

How do I get L0 and \alpha ?

**Comment by ericjb** [comment, marked as answer]
I found how to do this:

```
from sktime.forecasting.ets import AutoETS
fc = AutoETS(auto=False, trend=None, seasonal=None, n_jobs=-1)
fc.fit(y)
alpha = fc._fitted_forecaster.smoothing_level
L0 = fc._fitted_forecaster.initial_level
```

**Reply by fkiraly** [reply]
Hm, I did not know that!

I think we should write them to `self`, so `get_fitted_params` sees them.

I have opened an issue with a recipe: https://github.com/sktime/sktime/issues/7197
